esmvaltool/diag_scripts/autoassess/_plot_mo_metrics.py

```diff
@@ -111,22 +87,24 @@ def write_order_metrics(csvfile, metrics):
 
 
 def write_model_metrics(csvfile, metrics):
-    '''
-    Routine to write out model metrics csv file
+    """
+    Write out ordered model metrics.
+
+    Routine to write out model metrics csv file.
 
     An unordered list of metrics with a single value metric that are obtained
     from processing model output. Note that the model uncertainty also fits
     this description. This should be read in and out of a dictionary object
     with metric name as key and single float as value.
 
     :param str csvfile: CSV file name
-    :param dict metrics: Dictionary containing metric values
-    '''
+    :param dict metrics: Dictionary containing metric values.
+    """
     if metrics:
         try:
             outf = open(csvfile, 'w')
-        except IOError as e:
-            if e.errno == errno.EACCES:
+        except IOError as shit:
```

Comment:
Could you please use more professional names (also in other places)? We would not want anyone to feel offended when the read the esmvaltool source code.

Reply:
cheers @bouweandela this is a leftover from testing, sure thing will replace it :grinning: